### PR TITLE
Refactor about:passwords with Aphrodite

### LINKED ISSUE #7335: Refactor about:passwords with Aphrodite
## Test plan

https://github.com/brave/browser-laptop/pull/7336#issue-209135220

----

Refactor about:passwords with Aphrodite

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
Closes #7335

- passwords.less is removed
- PasswordsTh, PasswordsTr (HeadTr), and PasswordsTd (ActionsTd) are introduced
- classes are replaced with data-test-id

Auditors:

Test Plan:
1. Remember a password by logging in GitHub
2. Open about:passwords
3. Make sure styling is not broken

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

### CODE REVIEW

File: js/about/passwords.js [hunk 1]
```diff
@@ -23,13 +66,17 @@ class SiteItem extends React.Component {
...
-      <td className='passwordActions'>
-        <span className='passwordAction fa fa-times' title='Remove site'
-          onClick={this.onDelete} />
-      </td>
-      <td className='passwordOrigin'>{this.props.site}</td>
-    </tr>
+    return <PasswordsTr data-test-id='passwordItem'>
+      <ActionsTd>
+        <span className={css(styles.passwordAction)}>
+          <span className='fa fa-times' title='Remove site'
```
Comment (cezaraugusto) [inline js/about/passwords.js]: can we go ahead on this and localize title? change it to `data-l10n-id='removeSite'` and then on `passwords.properties` add 

```
removeSite.title=Remove site
```

Also add `fa fa-times` icon to `global.js` under [`appIcons`](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/styles/global.js#L159) property.

File: js/about/passwords.js [hunk 2]
```diff
@@ -84,21 +131,29 @@ class PasswordItem extends React.Component {
...
-        <span className='passwordAction fa fa-times' title='Delete password'
-          onClick={this.onDelete} />
-      </td>
-      <td className='passwordOrigin'>{password.get('origin')}</td>
-      <td className='passwordUsername'>{password.get('username')}</td>
-      <td className='passwordPlaintext'>
+    return <PasswordsTr data-test-id='passwordItem'>
+      <ActionsTd data-test-id='passwordActions'>
+        <span className={css(styles.passwordAction)}>
+          <span className='fa fa-times' title='Delete password'
```
Comment (cezaraugusto) [inline js/about/passwords.js]: ditto for icon class

File: js/about/passwords.js [hunk 3]
```diff
@@ -84,21 +131,29 @@ class PasswordItem extends React.Component {
...
-      </td>
-      <td className='passwordActions'>
-        <span className='passwordAction fa fa-clipboard' title='Copy password to clipboard'
-          onClick={this.onCopy} />
-      </td>
-    </tr>
+      </PasswordsTd>
+      <ActionsTd data-test-id='passwordActions'>
+        <span className={css(styles.passwordAction)}>
+          <span className='fa fa-clipboard' title='Copy password to clipboard'
```
Comment (cezaraugusto) [inline js/about/passwords.js]: same as first comment regarding icon and title

File: less/about/passwords.less [hunk 4]
```diff
@@ -1,72 +0,0 @@
...
-
-.passwordsPageFooter {
-  padding: 10px;
-  margin-bottom: 20px;
-  span {
-    color: grey;
-    cursor: pointer;
-    text-decoration: underline;
-  }
-}
```
Comment (cezaraugusto) [inline less/about/passwords.less]: super cool to see LESS files removed ++

File: test/components/notificationBarTest.js [hunk 5]
```diff
@@ -116,7 +116,7 @@ describe('notificationBar', function () {
       }).click('button=Yes')
       .tabByIndex(0)
       .loadUrl('about:passwords')
-      .waitForExist('tr.passwordItem')
+      .waitForExist('.passwordItem')
```
Comment (cezaraugusto) [inline test/components/notificationBarTest.js]: it should be `.waitForExist('[data-test-id="passwordItem"]')`

You can check if it works by running `npm run watch-test` and on other terminal tab run
```
npm run test -- --grep="autofills remembered password on login form"
```

File: js/about/passwords.js (hunk 1 above)
Comment (luixxiul) [inline js/about/passwords.js]: > can we go ahead on this and localize title?

I noticed but did not fix as I thought it is another issue. It should be handled with a follow up task.

Comment (cezaraugusto) [inline js/about/passwords.js]: np either way is fine

**Comment by luixxiul** [comment]
@cezaraugusto I added the update, so please review again. If it seems OK to you, I will squash the commits before merging. Thanks!

**Pull request merged**